src/main/c/org_apache_activemq_artemis_nativo_jlibaio_LibaioContext.c

```diff
@@ -502,7 +502,7 @@ static inline void iocb_destroy_bounded(struct io_control * theControl, int uppe
  * @param theControl
  */
 static inline void iocb_destroy(struct io_control * theControl) {
-    iocb_destroy_bounded(theControl, theControl->queueSize);
```

Comment:
it should really be the queueSize here for correctness... at this point queueSize should be == available though, in which case you could assert.